docs/source/conf.py

```diff
@@ -130,30 +104,54 @@
 # Subclasses should show parent classes docstrings if they don't override them.
 autodoc_inherit_docstrings = True
 
-# -- Options for linkcode extension ------------------------------------------
-
-
-# def linkcode_resolve(domain, info):
-#     if domain != "py":
-#         return None
-#     if not info["module"]:
-#         return None
-#     module_path = info["module"].replace(".", "/")
-#     # If only one `.`, assume it's a package
-#     if info["module"].count(".") == 1:
-#         return f"https://github.com/cgnorthcutt/cleanlab/tree/master/{module_path}"
-#     # Otherwise, it's a module
-#     else:
-#         return f"https://github.com/cgnorthcutt/cleanlab/blob/master/{module_path}.py"
-
-# # -- Exclude PyTorch methods -------------------------------------------------
-# def skip_torch_module_member(app, what, name, obj, skip, options):
-#     skip_torch = "Module." in str(obj) and name in dir(torch.nn.Module)
-#     if name == "dump_patches":  # Special handling for documented attrib
-#         skip_torch = True
-#     return skip or skip_torch
-
-
-# # -- Run setup ---------------------------------------------------------------
-# def setup(app):
-#     app.connect("autodoc-skip-member", skip_torch_module_member)
+# -- nbsphinx Configuration ---------------------------------------------------
+
+# This is processed by Jinja2 and inserted before each notebook
+nbsphinx_prolog = """
+{% set docname = env.doc2path(env.docname, base=None) %}
+
+.. raw:: html
+
+    <style>
+        .nbinput .prompt,
+        .nboutput .prompt {
+            display: none;
+        }
+    </style>
+
+    <p>
+        <a style= "background-color:white;color:black;padding:4px 12px;text-decoration:none;display:inline-block;border-radius:8px;box-shadow:0 2px 4px 0 rgba(0, 0, 0, 0.2), 0 3px 10px 0 rgba(0, 0, 0, 0.19)" href="https://colab.research.google.com/github/cleanlab/cleanlab/blob/master/docs/source/{{ docname|e }}" target="_blank">
+            <img src="https://colab.research.google.com/img/colab_favicon_256px.png" alt="Google Colab Logo" style="width:40px;height:40px;vertical-align:middle">   
+            <span style="vertical-align:middle">Run in Google Colab</span>
+        </a>
+    </p>
+"""
+
+# Change this to "always" before running in the doc's CI/CD server
+nbsphinx_execute = "always"
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = "furo"
+html_favicon = "_static/logo.png"
+html_title = "cleanlab Docs"
```

Comment:
Can we use this title instead: "cleanlab documentation"

Reply:
or just "cleanlab docs" for the sidebar (but D looks weird capitalized).

Reply:
done! changed to "cleanlab docs"